Metricas/Projetos/Teste/Maven Quick Start Archetype.urm.puml

```diff
@@ -0,0 +1,2 @@
+@startuml
```

Comment:
Tira essa pasta `Teste`, mas mantém `Projetos` pra gente saber onde colocar quando for rodar